contents/handbook/people/compensation.mdx

```diff
@@ -92,7 +92,7 @@ Whilst the terms of options for _any company_ could vary if we were ever acquire
 -   Vesting starts from your start date (not after a "probation period" or similar)
 -   For UK-based team members, our options are part of the [EMI share options scheme](https://www.gov.uk/tax-employee-share-schemes/enterprise-management-incentives-emis), which is tax-advantaged
 
-It can take time to approve options, as it requires a board meeting and company valuation. We can clarify the likely time frame at the time we're hiring you. In any case, you will not be disadvantaged as vesting will always start from when you joined PostHog.
+It can take time to approve options, as it requires a board meeting and company valuation. We can clarify the likely time frame at the time we're hiring you. In any case, you will not be disadvantaged as vesting will always start from when you joined PostHog. While we can commite to a particular percentage, we cannot commit to a particular strike price when offering share options, as the valuations are done by a third party and can vary depending on where we are in our funding cycle. 
```

Comment:
"commite"

Reply:
You know, when two mites work um together...? Fixing!